src/components/stepper/themes/step/light/step.base.scss

```
@@ -0,0 +1,446 @@
+        /* stylelint-disable */
+        display: var(--hide-horizontal-separator, var(--separator-disply-not-full, var(--horizontal-separator-display--first-of-type)));
+        /* stylelint-enable */
+    }
+
+    &::after {
+        inset-inline-end: 0;
+        /* stylelint-disable */
+        display: var(--hide-horizontal-separator, var(--separator-disply-not-full, var(--horizontal-separator-display--last-of-type)));
+        /* stylelint-enable */
+    }
+
+    &:focus,
+    &:hover {
+        cursor: pointer;
+        outline: none;
+    }
+
+    &:hover {
+        &::before,
+        &::after {
+            visibility: var(--header-separator-visibility--hover);
+        }
+
+        background: var(--header-background--hover);
+    }
+
+    &:focus,
+    &:focus-within,
+    &:active {
+        &::before,
+        &::after {
+            visibility: var(--header-separator-visibility--hover);
+        }
+
+        background: var(--header-background--focus);
+        box-shadow: inset 0 0 0 rem(1px) var(--header-shadow);
+    }
+}
+
+[part~='body'] {
+    display: var(--vertical-body-disply--last-of-type, var(--vertical-body-disply, none));
+    grid-row-start: var(--body-top, 2);
+    grid-column: span #{var(--steps-count)};
+    margin-inline-start: calc(var(--body-indent--vertical) / 2);
+    padding-inline-start: var(--horizontal-body-padding, #{calc(var(--body-indent--vertical) / 2)});
+    padding-inline-end: var(--horizontal-body-padding, var(--vertical-body-padding));
+    padding-block-start: var(--horizontal-body-padding, var(--vertical-body-padding));
+    padding-block-end: var(--horizontal-body-padding, var(--vertical-body-padding));
+    z-index: 1;
+    position: relative;
+    color: color(gray, 900);
+
+    // Thous tow lines here are on purpose, they make sure that step body content
+    // does not get involved in grid width calculation. They fix a very weird bug.
+    // https://stackoverflow.com/questions/73856035/css-grid-columns-changing-size-with-long-content/73858982#73858982
+    width: 0;
+    min-width: 100%;
+
+    &::before {
+        display: var(--hide-last-separator, var(--hide-first-separator, var(--vertical-separator-visibility)));
+        content: '';
+        position: absolute;
+        inset-inline-start: calc((var(--separator-size) / 2) * -1);
+        height: 100%;
+        inset-block-start: 0 !important;
+        border-inline-start: var(--separator-size) var(--separator-type) var(--separator-color);
+    }
+}
+
+[part='content'] {
+    display: var(--horizontal-content-disply, none);
+}
+
+[part~='top'] {
+    align-items: var(--step-not-full-header-aligmnet, flex-end);
+
+    &::before,
+    &::after {
+        inset-block-end: #{var(--step-separator-position)};
+    }
+
+    [part~='header'] {
+        align-items: var(--align-items, flex-start);
+        flex-direction: column-reverse;
+
+        &::before,
+        &::after {
+            inset-block-end: #{var(--step-separator-position)};
```

Why do we interpolate this here?

```suggestion
            inset-block-end: var(--step-separator-position);
```